docs/products/clickhouse/howto/sql-user-defined-functions.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Use SQL user defined functions in Aiven for ClickHouse®
+sidebar_label: SQL UDFs
+---
+
+Use SQL user defined functions (UDFs) in Aiven for ClickHouse® to speed up your queries
+and optimize your appilication performance.
+
+ClickHouse allows you to define your own functions, called user defined functions (UDFs).
+Aiven for ClickHouse supports SQL UDFs. These are automatically replicated to all nodes in
+the cluster and contained in backups.
+
+:::note
+Aiven for ClickHouse doesn't support
+[executable UDFs](https://clickhouse.com/docs/en/sql-reference/functions/udf#executable-user-defined-functions)
+or other types of UDFs other than SQL UDFs.
+:::
+
+## Create UDF in SQL
+
+To create an SQL UDF in Aiven for ClickHouse, run the `CREATE FUNCTION` expression
+including function parameters, constants, operators, or other function calls.
+
+```sql
+CREATE FUNCTION name AS (parameter0, ...) -> expression
+```
+
+## Limitations
+
+- Name of your UDF needs to be unique among other functions.
+- Recursive functions are not supported.
+- All variables used by your UDF need to be defined in its parameter list.
+
+## Example of using SQL UDFs
+
```

Comment:
Would there be any value in adding some narrative here about the example?

Reply:
The feature is very simple and has been available in the upstream for a while so the CH users are familiar with it. Also the example is self-explanatory. Seems the narrative would be redundant here.